Pages/navbar.html

```diff
@@ -73,37 +64,37 @@
                 >
                     <a
                         href="/Pages/gen_donate.html"
-                        class="block px-4 py-2 text-gray-900 hover:text-black"
+                        class="block px-4 py-2 text-gray-900 text-xs md:text-sm lg:text-lg hover:text-black font-bold"
                         >General Donation</a
                     >
                     <a
                         href="/Pages/tax_credit_dono.html"
-                        class="block px-4 py-2 text-gray-900 hover:text-black"
+                        class="block px-4 py-2 text-gray-900 text-xs md:text-sm lg:text-lg hover:text-black font-bold"
                         >Tax Credit</a
                     >
                 </div>
-            </li>
+            </li> -->
             <li class="relative group">
                 <a
                     href="/index.html"
                     class="block p-2 md:p-4 text-xs md:text-sm lg:text-base text-black font-extrabold uppercase transition-all duration-200 ease-in-out hover:text-c3153a break-words whitespace-normal"
                 >
-                    Accomplishments
+                    <!-- Accomplishments
```

Comment:
This comment excludes some of the opening tags and is making the navbar appear funky for me with some HTML syntax errors and invisible buttons, did you mean to comment out the entire donate and accomplishments menus?